kratos/processes/tetrahedral_mesh_orientation_check.h

```diff
@@ -65,32 +78,31 @@ class TetrahedralMeshOrientationCheck: public Process
      * @param rModelPart The model part to check.
      * @param ThrowErrors If true, an error will be thrown if the input model part contains malformed elements or conditions.
      */
-    TetrahedralMeshOrientationCheck(ModelPart& rModelPart,
-                                    bool ThrowErrors,
-                                    Flags options = NOT_COMPUTE_NODAL_NORMALS | NOT_COMPUTE_CONDITION_NORMALS | NOT_ASSIGN_NEIGHBOUR_ELEMENTS_TO_CONDITIONS
-                                    ):
-        Process(),
-        mrModelPart(rModelPart),
-        mThrowErrors(ThrowErrors), //to be changed to a flag
-        mrOptions(options)
+    TetrahedralMeshOrientationCheck(
+        ModelPart& rModelPart,
+        bool ThrowErrors,
+        Flags options = NOT_COMPUTE_NODAL_NORMALS | NOT_COMPUTE_CONDITION_NORMALS | NOT_ASSIGN_NEIGHBOUR_ELEMENTS_TO_CONDITIONS
```

Comment:
you should set NOT_ALLOW_CONDITIONS_WITH_SAME_GEOMETRY as default

Reply:
you did not change this. By default repeated conditions should not be allowed

Reply:
I will add it, but not being set gives the same result as being set false